packages/app/src/containers/ColorSwatchesInspector.tsx

```diff
@@ -0,0 +1,98 @@
+import { Selectors } from 'noya-state';
+import { rgbaToHex } from 'noya-colorpicker';
+import { Fragment, memo, useMemo } from 'react';
+import useShallowArray from '../hooks/useShallowArray';
+import ColorSelectRow from '../components/inspector/ColorSelectRow';
+import {
+  useApplicationState,
+  useSelector,
+} from '../contexts/ApplicationStateContext';
+import withSeparatorElements from '../utils/withSeparatorElements';
+import { Divider, Spacer } from 'noya-designsystem';
+import styled from 'styled-components';
+
+const Title = styled.div(({ theme }) => ({
+  ...theme.textStyles.small,
+  fontWeight: 'bold',
+  display: 'flex',
+  flexDirection: 'row',
+  userSelect: 'none',
+}));
+
+const Container = styled.div(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  padding: '10px',
+}));
+
+const ColorPickerInspector = memo(function ColorPickerInspector() {
+  const [state, dispatch] = useApplicationState();
+
+  const selectedSwatches = useShallowArray(
+    useSelector(Selectors.getSelectedColorSwatches),
+  );
+  const firstSwatch = selectedSwatches[0];
+
+  const firstColor = {
+    r: Math.round(firstSwatch.value.red * 255),
+    g: Math.round(firstSwatch.value.green * 255),
+    b: Math.round(firstSwatch.value.blue * 255),
+    a: firstSwatch.value.alpha,
+  };
+
+  let sameName = () =>
+    selectedSwatches.every((v) => v.name === firstSwatch.name);
+
+  const name =
```

Comment:
I would use `undefined` to represent multiple different names, and handle that specially within the `ColorSelectRow`

```ts
const name = selectedSwatches.length > 1 && !sameName() ? undefined : firstSwatch.name
```